src/main/java/com/educare/unitylend/controller/UserController.java

```diff
@@ -30,9 +27,41 @@ public List<User> getAllUsers() throws ControllerException {
         try {
             List<User> userList = userService.getUsers();
             return userList;
-        } catch (ServiceException e) {
+        } catch (Exception e) {
             log.error("Error encountered in getting the users");
             throw new ControllerException("Error encountered in getting the users", e);
         }
     }
+
+    @PostMapping("/create-user")
+    public ResponseEntity<String> createUser(@RequestBody User user) throws ControllerException {
+        // Create the user
+        try {
+            userService.createUser(user);
```

Comment:
Add null check for params in every controller method, Wherever necessary

Reply:
Yeah it has been done.